cmd/skopeo/proxy.go

```diff
@@ -197,6 +203,34 @@ func (h *proxyHandler) Initialize(args []interface{}) (replyBuf, error) {
 // OpenImage accepts a string image reference i.e. TRANSPORT:REF - like `skopeo copy`.
 // The return value is an opaque integer handle.
 func (h *proxyHandler) OpenImage(args []interface{}) (replyBuf, error) {
+	return h.openImageImpl(args, false)
+}
+
+// isManifestUnknownError is a copy of code from containers/image,
+// please update there first.
+func isManifestUnknownError(err error) bool {
```

Comment:
```suggestion
func isDockerManifestUnknownError(err error) bool {
```
or update the comment to point out that this is intended to handle errors from the c/image/docker transport.

---

(Implementation of this LGTM.)